pkg/spireentry/reconciler_test.go

```diff
@@ -75,3 +76,29 @@ func TestMakeEntryKey(t *testing.T) {
 		require.Equal(t, makeEntryKey(a), makeEntryKey(b))
 	})
 }
+
+func TestIsNamespaceIgnored(t *testing.T) {
+	type namespaceIgnoredTest struct {
+		ignoredNamespaces []*regexp.Regexp
+		namespace         string
+		expected          bool
+	}
+
+	regexp1, _ := regexp.Compile("s([a-z]+)re")
+	regexp2, _ := regexp.Compile("default")
+	regexps := []*regexp.Regexp{regexp1, regexp2}
+
+	var namespaceIgnoredTests = []namespaceIgnoredTest{
+		{regexps, "spire", true},
+		{regexps, "default", true},
+		{regexps, "spiffe", false},
+		{regexps, "kubernetes", false},
+	}
+
+	for _, test := range namespaceIgnoredTests {
+		actual := isNamespaceIgnored(test.ignoredNamespaces, test.namespace)
+		if actual != test.expected {
+			t.Errorf("isNamespaceIgnored(%s, %s): expected %t, actual %t", test.ignoredNamespaces, test.namespace, test.expected, actual)
+		}
```

Comment:
```suggestion
                require.Equal(t, test.expected, actual)
```

Reply:
Can also add a message. see https://pkg.go.dev/github.com/stretchr/testify/require#Equal and https://pkg.go.dev/github.com/stretchr/testify/require#Equalf